modules/script/script-impl/src/test/resources/myapplication/require-test.js

```diff
@@ -0,0 +1,29 @@
+const innerLib = require('inner-script');
+
+console.log('I am a require function');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
console.log("I am a require function");
```